parse_meddp.py

```diff
@@ -10,8 +13,20 @@ def parse_meddp(message: MessageIO):
     """
     # Implement the parsing logic to extract the relevant fields from the
     # message, according to the protocol specification in README.txt
+    return MeddpParser().parse_message_stream(message)
 
-    raise NotImplementedError()
+
+def print_messages(messages: MessageDict):
+    for key in messages:
```

Comment:
Seems like you don't really have any use for `key`.
```suggestion
    for message in messages.values():
```